jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/TransactionResource.java

```diff
@@ -172,12 +173,16 @@ public Response deleteCustomFields(@PathParam(ID_PARAM_NAME) final String id,
     @ApiResponses(value = {@ApiResponse(code = 400, message = "Invalid transaction id supplied"),
                            @ApiResponse(code = 404, message = "Invoice not found")})
     public Response getTags(@PathParam(ID_PARAM_NAME) final String id,
+                            @QueryParam(QUERY_WITH_PLUGIN_INFO) @DefaultValue("false") final Boolean withPluginInfo,
+                            @QueryParam(QUERY_WITH_ATTEMPTS) @DefaultValue("false") final Boolean withAttempts,
+                            @QueryParam(QUERY_PLUGIN_PROPERTY) final List<String> pluginPropertiesString,
                             @QueryParam(QUERY_AUDIT) @DefaultValue("NONE") final AuditMode auditMode,
```

Comment:
I don't think we need any of these properties: Those are for retrieving payment  and here we only care about retrieving tags (fetching the payment is just there to retrieve the `paymentId`).